src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable jsx-a11y/media-has-caption */
 import React, {useRef} from 'react';
 import PropTypes from 'prop-types';
-import {Badge} from '@momentum-ui/react';
 import Icon from '../generic/Icon/Icon';
 import Spinner from '../generic/Spinner/Spinner';
+import Badge from '../generic/Badge/Badge';
```

Comment:
this line would be better placed on its old line (line 4), for alpha order reasons.